src/index.html

```diff
@@ -11,12 +11,98 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
+    <header class="header-container">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li>
+            <a
+              href="#"
+              class="nav-item is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-item"
+            >
+              samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-item"
+            >
+              smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-item"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-item"
+            >
+              gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-item"
+            >
+              tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-item"
+            >
+              photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav-item"
+            >
+              video
+            </a>
```

Comment:
All `<a>` tags in the navigation menu have `href` attributes set to `#`. This is typically used as a placeholder. Ensure these links are updated to point to the correct URLs for each navigation item.